README.md

```diff
@@ -0,0 +1,15 @@
+# Blocktorch Debugger CLI Tool
+
+First, install the npm package:
+```bash
+npm i -g @blocktorch/debugger
+```
+
+To open the debugger, run the following command:
+```bash
+bt-debugger -tx <transactionHash>
```

Comment:
let's change the command to:
```suggestion
blocktorch debug -tx <transactionHash>
```

Reply:
Changed to `blocktorch debug <transactionHash>` in [22bcefc](https://github.com/blocktorch-xyz/debugger-cli/pull/1/commits/22bcefc6ec381a5d820febeabdd66871d7aed880)